hw/top_englishbreakfast/syn/top_englishbreakfast_syn_cfg.hjson

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
```

Comment:
same with this one..

feels like we should probably break this into a core portion and a top name portion or something

Reply:
I've discussed this with Michael and we agreed that we shouldn't make the top name a variable here. Most importantly, because we are not planning to do ASIC synthesis for secondary top-levels. This file got remove now.